org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/ast/InstanceOfExpression.java

```diff
@@ -273,19 +231,6 @@ public boolean containsPatternVariable() {
 public LocalDeclaration getPatternVariable() {
 	return this.elementVariable;
 }
-private void addSecretInstanceOfPatternExpressionValue(BlockScope scope1) {
-	LocalVariableBinding local =
-			new LocalVariableBinding(
-				InstanceOfExpression.SECRET_INSTANCEOF_PATTERN_EXPRESSION_VALUE,
-				TypeBinding.wellKnownType(scope1, T_JavaLangObject),
-				ClassFileConstants.AccDefault,
-				false);
-	local.setConstant(Constant.NotAConstant);
-	local.useFlag = LocalVariableBinding.USED;
-	scope1.addLocalVariable(local);
-	this.secretInstanceOfPatternExpressionValue = local;
-}
-
```

Comment:
This is inlined into the single call site. Don't see value in introducing a method for this purpose